tools/report-converter/tests/unit/smatch_output_test_files/sample.expected.plist

```diff
@@ -0,0 +1,69 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE plist PUBLIC "-//Apple//DTD PLIST 1.0//EN" "http://www.apple.com/DTDs/PropertyList-1.0.dtd">
+<plist version="1.0">
+<dict>
+	<key>diagnostics</key>
+	<array>
+		<dict>
+			<key>category</key>
+			<string>unknown</string>
+			<key>check_name</key>
+			<string>smatch</string>
```

Comment:
Are there any rule/checker names in smatch which could be used here?

Reply:
No, as far as we know, there is no rule/checker name in the smatch output and no option to get that.
I have reached out to Dan Carpenter, the smatch main developer, with that request:
https://lists.elisa.tech/g/development-process/topic/77462271

Reply:
Yess we are looking into it and once we find a solution we will integrate it into the parser, probably a seperate pull request like #2955 ?